milestone-3/frontend/src/components/recipePage.tsx

```diff
@@ -1,36 +1,47 @@
 import { Link } from "react-router-dom";
+import RecipePreview from "./recipePreview";
+import recipes from "../recipeData.json";
 
 interface Recipe {
     name: string;
     image: string;
-    desc: string;
+    description: string;
     ingredients: string [];
     instructions: string [];
 }
 
 function RecipePage(props: Recipe) {
     return (
-    <div className="sub-section">
+    <div>
     <h2>
         <Link to={`/recipes/${props.name}`} className="text">{props.name}</Link>
     </h2>
```

Comment:
This link is broken, it goes to /recipes/... instead of /recipePage/..., make sure you're consistent with the naming. Realistically, we don't need this to be a link because it just leads to the page we're already on. Instead, a header tag like h1 or h2 would be more appropriate.